system_notification/infra/http/server/fastapi_http_server.py

```
@@ -0,0 +1,61 @@
+    query_params: Dict[Any, Any] = {}
+    for key, value in request.query_params.items():
+        values_as_list = request.query_params.getlist(key)
+        if len(values_as_list) > 1:
+            query_params[key] = values_as_list
+            continue
+        query_params[key] = value
+    return query_params
+
+
+class FastApiHttpServer:
+    def __init__(self, server=uvicorn, app=FastAPI()) -> None:
```

Ruff (0.13.3)

Do not perform function call `FastAPI` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)